webpay/pay/tasks.py

```diff
@@ -370,6 +368,31 @@ def simulate_notify(issuer_key, pay_request, trans_uuid=None, **kw):
             simulated=sim_flag, task_args=[issuer_key, pay_request])
 
 
+@task(**notify_kw)
+@use_master
+def free_notify(notes, buyer_email, **kw):
+    """
+    Post JWT notice to an app about a pricePoint 0 product.
+
+    This isn't really much different from a regular notice except
+    that a fake transaction_uuid is created because solitude isn't
+    involved at all.
+
+    """
+
+    # No real transaction is created as this is a free product.
+    trans = {
+        'uuid': 'free:%s' % uuid.uuid4(),
+        'notes': notes,
+        'amount': '0.00',
+        'currency': 'USD',
+        'type': constants.TYPE_PAYMENT
+    }
+    extra_response = {'buyer_email': buyer_email}
```

Comment:
hrrmmm. We may not want to expose emails to all third party apps who offer in-app purchases at the free tier. One alternative could be to create a solitude buyer and pass ``buyer_uuid`` and let Zamboni look up the email in Solitude. I don't know if it's worth the extra effort for that, just a thought.